docs/docs/guides/tutorial/rill-basics/5-dashboard.md

```diff
@@ -42,15 +43,15 @@ measures: '*'
 
 For a quick summary on the different components of an explore dashboard, and its respective parts in the dashboard UI.
 
-<img src = '/img/tutorials/103/simple-dashboard.gif' class='rounded-gif' />
+<img src = '/img/tutorials/rill_basics/simple-dashboard.gif' class='rounded-gif' />
```

Comment:
For consistency, and to match the browser URL, should `rill_basics` be `rill-basics`?